### ISSUE TITLE
[Bug]: Cannot download and install new runners

### ISSUE THREAD

**Issue by dumbnerd08** [bottlesdevs/Bottles#2554]
### Describe the bug

When I try to download and install a new runner, it just gives me an error.

### To Reproduce

1. Open bottles
2. Click on the hamburger menu in the top right
3. Click preferences
4. Click runners
5. Click on caffe (or any runner category, i have tried reproduce this issue on caffe and soda)
6. Click the download button on any version (caffe 7.20 in my case)
7. Observe it erroring nearly immediately
8. Try again
9. Give up

### Package

Flatpak from Flathub

### Distribution

Ubuntu 22.04.1

### Troubleshooting Logs

```shell
Official Package: true
Version: '50.1'
DE/WM: ubuntu
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: false
Graphics:
    vendors:
        nvidia: &amp;id002
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/nvidia_icd.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/nvidia_icd.json
            nvngx_path: /usr/lib/x86_64-linux-gnu/GL/nvidia-525-60-11/extra/nvidia/wine
        intel: &amp;id001
            vendor: intel
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.i686.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json
    prime:
        integrated: *id001
        discrete: *id002
Kernel:
    Type: Linux
    Version: 5.15.0-58-generic
Disk:
    Total: 8305676288
    Free: 8305508352
RAM:
    MemTotal: 15.5GiB
    MemAvailable: 11.3GiB
Bottles_envs: null
```

**Comment by jshsakura** [comment]
Same for me

**Comment by Kinsteen** [comment]
Hi, can you launch bottles from the command line (`flatpak run com.usebottles.bottles`) and send the logs here please?

**Comment by Neddie67** [comment]
neddie@pop-os:~$ flatpak run com.usebottles.bottles
04:02:05 (INFO) Bottles Started! 
04:02:09 (INFO) Performing Bottles checks… 
04:02:09 (INFO) Dxvks found:
	 - dxvk-2.0-1-196fefe
	 - dxvk-async-2.0
 
04:02:09 (INFO) Vkd3ds found:
	 - vkd3d-proton-2.8-1-f09e31a
 
04:02:09 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.6-1-0827efb
 
04:02:09 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.1
 
04:02:09 (INFO) Runners found:
	 - sys-wine-7.0.1
	 - wine-ge-proton7-34
	 - soda-7.0-8
 
04:02:09 (INFO) Bottles found:
	 - blizzard
 

(bottles:2): Gtk-CRITICAL **: 04:02:09.357: gtk_widget_set_parent: assertion '_gtk_widget_get_parent (widget) == NULL' failed
04:02:09 (INFO) Library saved 
04:02:11 (INFO) Catalog installers loaded 
04:02:11 (INFO) Catalog dependencies loaded 
04:02:11 (INFO) Catalog components loaded 
04:02:16 (INFO) Installing component: [dxvk-1.10.3]. 
dxvk-1.10.3.tar.gz (100%) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ (8.5 MB/8.5 MB - 5.2 MB)

04:02:19 (INFO) Renaming [dxvk-1.10.3.tar.gz] to [dxvk-1.10.3.tar.gz]. 
04:02:19 (INFO) Dxvks found:
	 - dxvk-2.0-1-196fefe
	 - dxvk-1.10.3
	 - dxvk-async-2.0
 
04:02:19 (INFO) Component installed: dxvk dxvk-1.10.3 
04:02:31 (INFO) Installing component: [dxvk-1.10.2]. 
dxvk-1.10.2.tar.gz (100%) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ (8.5 MB/8.4 MB - 3.2 MB)

04:02:35 (INFO) Renaming [dxvk-1.10.2.tar.gz] to [dxvk-1.10.2.tar.gz]. 
04:02:35 (INFO) Dxvks found:
	 - dxvk-2.0-1-196fefe
	 - dxvk-1.10.3
	 - dxvk-1.10.2
	 - dxvk-async-2.0
 
04:02:35 (INFO) Component installed: dxvk dxvk-1.10.2 
04:02:36 (INFO) Opening the file manager in the path … 
neddie@pop-os:~$ flatpak run com.usebottles.bottles
04:03:09 (INFO) Bottles Started! 
04:03:13 (INFO) Performing Bottles checks… 
04:03:13 (INFO) Dxvks found:
	 - dxvk-2.0-1-196fefe
	 - dxvk-1.10.3
	 - dxvk-async-2.0
 
04:03:13 (INFO) Vkd3ds found:
	 - vkd3d-proton-2.8-1-f09e31a
 
04:03:13 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.6-1-0827efb
 
04:03:13 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.1
 
04:03:13 (INFO) Runners found:
	 - sys-wine-7.0.1
	 - wine-ge-proton7-34
	 - soda-7.0-8
 
04:03:13 (INFO) Bottles found:
	 - blizzard
 

(bottles:2): Gtk-CRITICAL **: 04:03:13.249: gtk_widget_set_parent: assertion '_gtk_widget_get_parent (widget) == NULL' failed
04:03:13 (INFO) Library saved 
04:03:15 (INFO) Catalog installers loaded 
04:03:15 (INFO) Catalog dependencies loaded 
04:03:15 (INFO) Catalog components loaded 
04:03:20 (INFO) Installing component: [dxvk-1.10.1]. 
04:03:20 (WARNING) File [dxvk-1.10.1.tar.gz] already exists in temp, skipping. 
04:03:20 (ERROR) Downloaded file [dxvk-1.10.1.tar.gz] looks corrupted. 
04:03:20 (ERROR) Source cksum: [08c0354ef2d2d22901545dcb153c618c] downloaded: [a1be025ba0542741f47e66d3efb6c73e] 
04:03:21 (ERROR) Removing corrupted file [dxvk-1.10.1.tar.gz]. 
04:03:21 (ERROR) Task  failed 
04:03:21 (ERROR) Task  failed 
04:03:41 (INFO) Dxvks found:
	 - dxvk-2.0-1-196fefe
	 - dxvk-async-2.0
 
04:03:41 (INFO) Component uninstalled: dxvk dxvk-1.10.3 
04:03:46 (INFO) Installing component: [dxvk-1.10.1]. 
dxvk-1.10.1.tar.gz (100%) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ (8.3 MB/8.3 MB - 5.2 MB)

04:03:49 (INFO) Renaming [dxvk-1.10.1.tar.gz] to [dxvk-1.10.1.tar.gz]. 
04:03:49 (INFO) Dxvks found:
	 - dxvk-1.10.1
	 - dxvk-2.0-1-196fefe
	 - dxvk-async-2.0
 
04:03:49 (INFO) Component installed: dxvk dxvk-1.10.1 




having the same issue here where if i launch the app normally try and download any runner or dll component like example i tried to download dxvk normally without terminal launch, it fails so i then launch it with the (flatpak run com.usebottles.bottles) and it seems to find the file that was downloaded before and it thinks its corrupted then deletes it. i try to download it again and installs fine through the terminal.

**Comment by swsnr** [comment]
I've the same problem with dependencies: When I start the bottles flatpak from Gnome, the `dotnet48` download fails.

* `journalctl --user --unit=app-flatpak-com.usebottles.bottles-15169.scope --output=short-iso --no-hostname` (slightly redacted for privacy): https://gist.github.com/swsnr/6b2cefb53eb15fc6b20966b5d59a1197

"Download failed" is all I see as an error message, and the user interface shows a "failed icon" beneath the dependency name.

When I immediately retry the same from `flatpak run com.usebottles.bottles`  the dependency succeeds: 

* `flatpak run com.usebottles.bottles`: https://gist.github.com/swsnr/2c57949b8d728e89da8a8a41afecb271

**Comment by Kinsteen** [comment]
Thanks for all of your reports. The bug has been located and a fix will be available in a few days.

For a workaround, run Bottles from the command line: `flatpak run com.usebottles.bottles`

**Comment by Neddie67** [comment]
> Thanks for all of your reports. The bug has been located and a fix will be available in a few days.
> 
> For a workaround, run Bottles from the command line: `flatpak run com.usebottles.bottles`

That's awesome thank you. I love all the work your team Is bringing to this app and can't wait for more improvements even tho the app is already pretty damn perfect 😭

**Comment by Kinsteen** [comment]
This is fixed in Bottles 50.2, which will be released in a few hours to a day from now.
